server/views/partials/additionalPssConditionsSummary.njk

```diff
@@ -17,7 +17,7 @@
 
     <div class="govuk-grid-row check-answers-header">
         <div class="govuk-grid-column-three-quarters">
-            <h2 id="additional-pss-conditions-heading" class="govuk-heading-m">{{ title or 'Additional post sentence supervision requirements' }}</h2>
+            <h2 id="additional-pss-conditions-heading" class="govuk-heading-m">{{ title or 'Additional post sentence requirements details' }}</h2>
```

Comment:
If this is the wording in the prototype, I'd double-check this with Tom as PSS stands for Post Sentence Supervision, and removing the 'Supervision' seems odd

Reply:
Ok I will check with Tom.

Reply:
![image](https://github.com/ministryofjustice/create-and-vary-a-licence/assets/129746957/8d15e7f8-c5df-43b9-b505-2176080657b3)
I checked with Tom and the heading should be as in prototype itseems.